### PR TITLE
fix(insights): Fix property filter group and expression UI

### PR DESCRIPTION
**Pull request by Twixes**
## Problem

This was a bit ugly:

| Property filter groups | HogQL expression |
|--------|--------|
| before-group | before-hog | 

## Changes

This looks better:

| Property filter groups | HogQL expression |
|--------|--------|
|  after-group |  after-hog |

## How did you test this code?

Added UI snapshots for insight editors.

### CODE REVIEW

**Comment by Twixes** [comment]
Ahh, Storybook only shows legacy UI, not data exploration… Do you think data exploration is in a state where all of Storybook can have that flag on, or should we still have both branches covered @thmsobrmlr?

**Comment by thmsobrmlr** [comment]
> Ahh, Storybook only shows legacy UI, not data exploration… Do you think data exploration is in a state where all of Storybook can have that flag on, or should we still have both branches covered @thmsobrmlr?

Yup, it's coming here https://github.com/PostHog/posthog/pull/15497. The PR should be ready to go apart from a Jest test that I need to fix and go in on Tuesday. Took a while to get this ready since "simply toggling the flag" affected a good chunk of tests and I discovered some issues with Experiments and Notebooks.

**Review by thmsobrmlr — APPROVED** [review]
✨

File: frontend/src/lib/components/PropertyFilters/PropertyFilters.scss [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
+    .PropertyFilters__content {
         flex: 1;
         display: flex;
         flex-wrap: wrap;
-        gap: 0.5rem;
         overflow: hidden;
     }
 
-    .PropertyFilters-prefix {
+    .PropertyFilters__pprefix {
```
[the remark this answers is not in the document]
Comment (Twixes) [reply frontend/src/lib/components/PropertyFilters/PropertyFilters.scss]: Fixed already!

**Comment by Twixes** [comment]
Looks like Firefox visual regression tests have a memory leak and I've found no way to fix this. :/ Will probably remove Firefox in a separate PR.

**Pull request merged**